src/SfxWeb/src/app/views/BaseView.ts

```diff
@@ -0,0 +1,5 @@
+import { ElementRef } from "@angular/core";
```

Comment:
is this still used?

Reply:
It's not being used, this was originally for the 'skip to content', I asked if we could keep it in case we wanted to use it later.